javalib/src/main/scala/java/util/Map.scala

```diff
@@ -1,5 +1,15 @@
+/*
+ * Ported from Scala.js
+ *   commit SHA1: f7be410
+ *   dated:  2020-10-12
```

Comment:
I think the extra indentation here is not really standard or perhaps put it on one line since it is not needed unless it is needed and it would make it less visible. I wouldn't change this unless other reviews make you change the file.

Reply:
Thank you for looking.  As you have experience, Scala.js and Scala Native have different native formatting styles.
We have an opportunity to develop some consistency but I think it will take generations.

I am using a somewhat-automated script and can change the header.  I chose the current one to run up
the flagpole because it _exactly_ (I believe & hope) matches the Scala.js copyright comment style it replaces.

I think I have since eliminated a blank immediately before the date, so that it "dated: 2020-mumble", one space.

Let's see what other people say.   Personally, I am more invested in _some_ consistent style, rather than any particular one.  I do find having the SHA and date of the origin to be almost essential for maintenance.   I have encountered 
that problem with Scala.js, Ryu IEEE floats, and the RE2 regex ports I have done.  Many a wasted day trying to re-establish a baseline, so that I could figure out what had changed since. 

IMHO, providing a bit of providence is the mark of a maturing project.

Reply:
May be we can automate the for the header,maybe we can make use of sbt-header plugin.

Reply:
Kerr, I took a quick look at sbt-header.  It looks like it handles licenses and a wide variety of them, but I did not see a way to do the transform needed here.  I will keep your suggestion in mind, in case I am more clever tomorrow...